ethdb/pebble/pebble.go

```diff
@@ -173,7 +173,7 @@ func (db *Database) Get(key []byte) ([]byte, error) {
 
 // Put inserts the given value into the key-value store.
 func (db *Database) Put(key []byte, value []byte) error {
-	return db.db.Set(key, value, nil)
+	return db.db.Set(key, value, &pebble.WriteOptions{Sync: false})
```

Comment:
Just saw that there's a `pebble.NoSync` constant too so you don't need to redeclare it every time. Also pls add to Delete too.

Reply:
writeOpt is useless for delete. So I don't add it.